TEveEventDisplay/src/TEveMu2eTracker.cc

```diff
@@ -130,7 +113,7 @@ namespace mu2e{
           CLHEP::Hep3Vector diskPos = calo->disk(idisk).geomInfo().origin() - _detSysOrigin;
           diskPos += CLHEP::Hep3Vector(0.0, 10000.0, (-holeDZ+frontPanelHalfThick));
           double k = diskOuterRailOut + diskInnerRingIn;
-          for(int ic=0; ic<30; ic++){
+          for(int ic=0; ic<calo->nCrystal()/45; ic++){
```

Comment:
Where does the number 45 come from?  This is integer arithmetic, by default it will round down, is that what you want?